config/maintainers.json

```diff
@@ -12,14 +12,14 @@
       "show_on_website": false
     },
     {
-      "alumnus": false,
+      "alumnus": true,
```

Comment:
[It should be `false`](https://github.com/exercism/ecmascript/pull/357#pullrequestreview-69654498).

Reply:
Should be fixed now